keyboards/keebio/iris/keymaps/radlinskii/config.h

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2017 Danny Nguyen <danny@keeb.io>
+Copyright 2022 Ignacy Radliński @radlinskii
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
```

Comment:
this piece of license notice was generated when creating keymap from the default `iris/rev6` keymap, but since I've changed this file, I've added line `3`, is it ok?